addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -108,6 +124,53 @@ func _drop_data(at_position: Vector2, data: Variant) -> void:
 	reconnect_block.emit(block)
 
 
+func _drop_obj_property(at_position: Vector2, data: Variant) -> void:
+	var obj_name = str(data.get("object", "")).get_slice(":", 0)
+	var prop_name = data.get("property", "")
```

Comment:
Nit: use descriptive variable names:
```suggestion
	var object_name = str(data.get("object", "")).get_slice(":", 0)
	var property_name = data.get("property", "")
```

Also, it is worth the extra safety? Can't you just use `data["object"]`, `data["property"]` here? In other words. is there any property that when dropped doesn't come with the "object" or "property" keys in the data?

Reply:
Have not noticed any property that doesn't come with the "object" or "property" keys in the data, yet.  Maybe, they can be accessed directly.

Reply:
Access the "object", "property" and "value" within the new commit directly.